--- phpwiki-1.3.12p2/configurator.php	2005-09-18 15:06:41.000000000 +0300
+++ /home/glen/configurator.php	2006-05-15 01:32:33.000000000 +0300
@@ -63,8 +63,8 @@
 $scriptname = str_replace('configurator.php', 'index.php', $HTTP_SERVER_VARS["SCRIPT_NAME"]);
 
 $tdwidth = 700;
-$config_file = (substr(PHP_OS,0,3) == 'WIN') ? 'config\\config.ini' : 'config/config.ini';
-$fs_config_file = dirname(__FILE__) . (substr(PHP_OS,0,3) == 'WIN' ? '\\' : '/') . $config_file;
+$config_file = (substr(PHP_OS,0,3) == 'WIN') ? 'config\\config.ini' : 'config.ini';
+$fs_config_file = '/etc/webapps'. (substr(PHP_OS,0,3) == 'WIN' ? '\\' : '/') . $config_file;
 if (isset($HTTP_POST_VARS['create']))  header('Location: '.$configurator.'?show=_part1&create=1#create');
 
 // helpers from lib/WikiUser/HttpAuth.php
--- phpwiki-1.3.12p2/index.php	2005-02-09 21:01:47.000000000 +0200
+++ /home/glen/index.php	2006-05-15 01:33:15.000000000 +0300
@@ -33,7 +33,7 @@
 rcs_id('$Id$');
 
 require_once(dirname(__FILE__).'/lib/IniConfig.php');
-IniConfig(dirname(__FILE__)."/config/config.ini");
+IniConfig("/etc/webapps/phpwiki/config.ini");
 
 ////////////////////////////////////////////////////////////////
 // PrettyWiki
--- phpwiki-1.3.12p2/lib/IniConfig.php	2006-02-19 21:41:38.000000000 +0200
+++ /home/glen/IniConfig.php	2006-05-15 01:38:53.000000000 +0300
@@ -188,7 +188,7 @@
          );
 
     $rs = @parse_ini_file($file);
-    $rsdef = @parse_ini_file(dirname(__FILE__)."/../config/config-default.ini");
+    $rsdef = @parse_ini_file('/etc/webapps/phpwiki/config-default.ini');
     foreach ($rsdef as $k => $v) {
     	if (defined($k)) {
     	    $rs[$k] = constant($k);
